swagger.yaml

```diff
@@ -412,6 +412,29 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  /geo/{year}/{geography_code}:
+    get:
+      operationId: GetGeo
+      tags:
+        - public
+      summary: Get geographic info about a region
```

Comment:
I'd avoid using 'region' here - as discussed offline, same reason as changing the path parameter name (FYI other readers - 'region' is a specific type of geography already). Maybe 'Get geographic info about an area'?

Reply:
done